cypress/e2e/felony/utils/testIneligiblePage.ts

```diff
@@ -0,0 +1,19 @@
+export default function testIneligiblePage(link: string) {
+  describe("f-classa-main-1-ineligble", () => {
+    it('has ineligibility content', () => {
+      cy.visit(`/calculator/${link}`);
+      cy.contains('Your conviction is not eligible to vacate')
+        .should('be.visible')
+      cy.contains('as defined in RCW 9.94A.030')
+        .should('have.attr', 'href', 'https://app.leg.wa.gov/RCW/default.aspx?cite=9.94A.030')
+      cy.contains('resources page')
+        .should('be.visible')
```

Comment:
```suggestion
```
If we are not test the `href` here, i would say that we can just remove that test, because, as we were talking, we dont want to be too much coupled on texts.

If you think that this test is important, i suggest to use `a` tag and check `href`.